snap/snapcraft.yaml

```diff
@@ -18,14 +18,11 @@ parts:
     source: .
     prime:
       - bin/tvhproxy.sh
+
   tvhproxy:
     plugin: python
     source: https://github.com/jkaberg/tvhProxy.git
     source-type: git
-    python-packages:
-      - flask
-      - requests
-      - gevent
+    requirements: requirements.txt
     install: |
-      cp tvhProxy.py $SNAPCRAFT_PART_INSTALL
-      cp -r templates $SNAPCRAFT_PART_INSTALL
+      cp  -R ./ $SNAPCRAFT_PART_INSTALL/opt/tvhproxy
```

Comment:
What is the reason to move it into $SNAP/opt?

Reply:
That was an aesthetic change to tidy up the filesystem.

Reply:
I would prefer, if we move it, to move it to `$SNAP/bin`. IMHO `/opt` doesn't make much sense inside a snap.